### ISSUE TITLE
Confused about "Reference does not exist" errors when using bundle option

### ISSUE THREAD

**Issue by StephenNay** [Redocly/openapi-cli#84]
My root file uses $ref to include all of the various OAS components, like this:

```
components:
  schemas: 
    $ref: ./schemas.yaml
  parameters:
    $ref: ./parameters.yaml
  securitySchemes: 
    $ref: ./securitySchemes.yaml
  responses:
    $ref: ./responses.yaml
  requestBodies:
    $ref: ./requestBodies.yaml
  examples: 
    $ref: ./examples.yaml
```

When I run openapi to bundle the files into a single file, it complains about unresolved references, like this:

```
[1] NetSense_API_v3.yaml:1029:11 at #/paths/['/login']/get/responses/default/$ref
Reference does not exist.

1027|                 $ref: '#/components/schemas/UserAuthorization'
1028|         default:
1029|           $ref: '#/components/responses/400-ErrorResponse2'
1030| 
1031|   /logout:
1032|     get:

Error was generated by resolve-ref rule.
```

In this case, 400-ErrorResponse2 _DOES_ exist in the responses.yaml file, so if openapi bundled the files together, the reference would be valid. 

Am I doing something wrong?  The equivalent commands in speccy and swagger-cli both work fine.

**Comment by adamaltman** [comment]
I think it's related to #79.

**Comment by adamaltman** [comment]
#79 was merged.  Can you please check it again?

**Comment by RomanHotsiy** [comment]
@adamaltman yes!

@StephenNay I just merged #79 and released `v0.7.0`. Your issue should be fixed now. Could you us know if this is fixed now? Thanks!

**Comment by RomanHotsiy** [comment]
Closing.

@StephenNay feel free to reopen in case this is still not resolved.
